android/src/main/java/com/bitmovin/player/reactnative/BufferModule.kt

```diff
@@ -32,36 +32,24 @@ class BufferModule(private val context: ReactApplicationContext) : ReactContextB
                 player.buffer.getLevel(bufferType, MediaType.Audio),
                 player.buffer.getLevel(bufferType, MediaType.Video),
             )
-            JsonConverter.fromRNBufferLevels(bufferLevels).let {
-                promise.resolve(it)
-            }
+            promise.resolve(bufferLevels.toJson())
```

Comment:
promise is already resolved by addUIBlock
```suggestion
            bufferLevels.toJson()
```

Reply:
Fixed: f2ff634e7045f1399b537de620e811bb58d94dc2